### PR TITLE
Build runproof and owise removal

### PR DESCRIPTION
**Pull request by nishantjr**

### CODE REVIEW

**Review by hjorthjort — APPROVED** [review]
LGT
Does this solve a prover issue you're having?

File: tests/proofs/dexter/dexter.md [hunk 1]
```diff
@@ -619,12 +658,12 @@ If the contract execution fails, storage is not updated.
...
  // -------------------------------------------------------------
   rule #TokenTransferType(false) => #Type(pair address pair address nat) [simplification]
-  rule #TokenTransferType(true)  => #Type(list pair address list pair address pair nat nat) [simplification]
+  rule #TokenTransferType(true)  => #Type(list (pair address (list (pair address (pair nat nat))))) [simplification]
 
   syntax Data ::= #TokenTransferData(Bool, Address, Address, Int, Int) [function, functional]
  // -----------------------------------------------------------------------------------------
-  rule #TokenTransferData(false, From, To, _TokenID, TokenAmt) =>   Pair From   Pair To              TokenAmt [simplification]
-  rule #TokenTransferData(true,  From, To,  TokenID, TokenAmt) => [ Pair From [ Pair To Pair TokenID TokenAmt ] ;; .InternalList ] ;; .InternalList [simplification]
+  rule #TokenTransferData(false, From, To, _TokenID, TokenAmt) =>   Pair From    Pair To              TokenAmt [simplification
```
Comment (hjorthjort) [inline tests/proofs/dexter/dexter.md:665]:
```suggestion
  rule #TokenTransferData(false, From, To, _TokenID, TokenAmt) =>   Pair From    Pair To              TokenAmt [simplification]
```

**Comment by nishantjr** [comment]
> LGT
> Does this solve a prover issue you're having?

Yes, the `owise` rule was causing theprover to crash

**Comment by hjorthjort** [comment]
Are you thinking we'll change the CI staging back after the project is done? If so, makes sense to me, but lets record it in an issue.

**Review by ehildenb — APPROVED** [review]

**Comment by nishantjr** [comment]
> Are you thinking we'll change the CI staging back after the project is done? If so, makes sense to me, but lets record it in an issue.

I think this change should be permanent. Because, in general most changes we make will be to the semantics and proofs.
Changes to Tezos, such as updating to a new version, or changing how the compatibility tests are run should be relatively rare.

**Pull request merged**